odoo/addons/base/tests/test_base.py

```diff
@@ -52,13 +61,268 @@ def test_05_safe_eval_forbiddon(self):
             safe_eval('open("/etc/passwd","r")')
 
         # no forbidden opcodes
-        with self.assertRaises(ValueError):
+        with self.assertRaises(SyntaxError):
             safe_eval("import odoo", mode="exec")
 
         # no dunder
-        with self.assertRaises(NameError):
+        with self.assertRaises(ValueError):
```

Comment:
Is it intended that this test is broken? we can actually access __ elements, the test is actually raising because we cannot manipulate to BaseCase type